### DISCUSSION TITLE
need to learn

### DISCUSSION THREAD

**Discussion by Amaaaan7** [community/community#145678] [New to GitHub] [answered]
i want to have some people that i can work on any kind of project . im open to help anyone in their repositories. i just want to learn .

**Comment by CyberSphinxxx** [comment, marked as answer]
Hey @Amaaaan7, That’s great! You can start by contributing to open-source projects that interest you. Look for issues to solve, help with documentation, or suggest improvements. GitHub is full of opportunities to learn and collaborate, so don’t hesitate to reach out or start your own project!

**Reply by Surealist123** [reply]
![file_000000001cd861f5a1c122161b3ea8ba (2)](https://github.com/user-attachments/assets/af967246-6439-45b1-b99c-2007db7978f6)
![file_0000000015c461f7be9067a61549c479](https://github.com/user-attachments/assets/b6cd9c23-6b07-4d4a-9674-a61d773966ef)
![file_00000000d1e861f592fa81625a134629](https://github.com/user-attachments/assets/71383852-2046-462e-b6eb-0483db65f40b)
![file_000000005fbc622f8f02c5011a81ed81 (2)](https://github.com/user-attachments/assets/78b2821c-f956-4959-8eb0-827d3f538d54)
![file_00000000de0061fb869d5cf1ca4356f1](https://github.com/user-attachments/assets/87df6a84-c994-4ea7-be59-2ce54d49eed5)
![file_00000000b9046230ab75567abe918ab8](https://github.com/user-attachments/assets/69b8e77e-ea9f-44df-b761-747d3c946f59)
![file_00000000f0146230a409eb67761f822b (2)](https://github.com/user-attachments/assets/8b121fdc-8b51-4663-a8f5-da0d99821152)
![file_000000009c0c61f7bd190249b04ad7da (3)](https://github.com/user-attachments/assets/6d61bce7-682b-4259-9d80-bd6a767b771d)
E
I am contacting you regarding a couple of projects I would love to get help with, I really want to start a company based around mental health awareness and development I get really confused on applications and would love to have an avatar that could point me in the right direction and I want to connect with people here. My Max. It is about 45 mins at a time when I'm working on computers. I want to create an application but I need to learn how to from the start

**Comment by Barmanji** [comment]
Hi **@Amaaaan7,**

That’s awesome to hear! I’m also looking to collaborate and learn by contributing to open-source projects. If you have any repositories or projects you’re working on, I’d love to help out in any way I can—whether it’s solving issues, improving documentation, or brainstorming new ideas.

Feel free to reach out or share any projects you’re passionate about. Let’s grow and learn together!

Looking forward to collaborating,
Ajay Barman

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @Amaaaan7 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
